src/Uploadcare/File.php

```diff
@@ -213,7 +213,19 @@ public function delete()
    */
   public function getUrl($postfix = null)
   {
-    $url = sprintf('%s/%s/', $this->api->getCdnUri(), $this->uuid);
+    $url = sprintf('%s/%s', $this->api->getCdnUri(), $this->getPath($postfix));
+    return $url;
+  }
+
+  /**
+   * Get local URL path of original image
+   *
+   * @param string $postfix
+   * @return string
+   */
+  public function getPath($postfix = null)
+  {
+    $url = sprintf('%s/', $this->uuid);
```

Comment:
according to https://tools.ietf.org/html/rfc3986#section-3.3
absolute path (and this is absolute path) starts with `/` :)

Reply:
fixed